docs/pages/dev-guide/developer-guide.md

```diff
@@ -819,6 +821,8 @@ pods:
         memory: 256
 ```
 
+Currently an empty YAML dictionary is passed as the body for each network definition under `networks`, since we only support joining virtual networks by name, but in the future it will be possible to specify port mappings and other information in a network definition.
```

Comment:
We really need some equivalent of a json schema for YAML so we can normalize this documentation.